services/s3/src/test/java/software/amazon/awssdk/services/s3/internal/resource/MultiRegionAccessPointEndpointResolutionTest.java

```diff
@@ -42,7 +42,7 @@ public class MultiRegionAccessPointEndpointResolutionTest {
 
     private final static String MULTI_REGION_ARN = "arn:aws:s3::123456789012:accesspoint:mfzwi23gnjvgw.mrap";
     private final static URI MULTI_REGION_ENDPOINT =
-        URI.create("https://mfzwi23gnjvgw.mrap.accesspoint.s3-global.amazonaws.com");
+        URI.create("https://mfzwi23gnjvgw.mrap.accesspoint.s3-global.amazonaws.com/");
```

Comment:
In the new code path the "path" part of the URI gets normalized to `"/"` if not present, thus we need to add the leading slash as well here.

Reply:
q: I'm guessing there is no backwards compatibility concern with this change. but curious is this caused by the new signers or something else?

Reply:
yes, is part of the `sanitize` that does ensure that the URI path is there. I hope that this bit will work fine with signing but as of now I'm can't tell with certainty.

Reply:
I remember seeing issues with trailing slash before. Is there any reason this is changed? Should it add trailing slash?

Reply:
Well, I don't know if it **should** but I know that it **does**, thus the test needed to be changed. I think is possible, albeit unlikely, that this might break something but we won't know until we start testing. Given that the logic that does sanitize the path from empty to a single slash is somewhere else (the signer logic) I'd rather park this and revisit at testing.

Reply:
Yeah, agreed that we should park this. Let's add a TODO though to revisit this